protocol/messenger_group_chat_test.go

```diff
@@ -409,3 +410,47 @@ func (s *MessengerGroupChatSuite) TestGroupChatHandleDeleteMemberMessage() {
 	defer s.NoError(admin.Shutdown())
 	defer s.NoError(member.Shutdown())
 }
+
+func (s *MessengerGroupChatSuite) TestGroupChatMembersRemoval2() {
```

Comment:
Should we rename this test to better describe the issue that was happening?

Reply:
oh yes, sorry, forgot to do that